core/data-api/src/main/java/com/dev/innerview/core/data_api/InnerViewRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.dev.innerview.core.data_api
+
+import com.dev.innerview.core.model.InnerView
+import com.dev.innerview.core.model.InnerViewType
+import kotlinx.coroutines.flow.Flow
+
+interface InnerViewRepository {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line
<!-- __reviewdog__:ChBmM2QxMGYzZTIwM2VjZDMyEgZrdGxpbnQ= -->